src/config/dynamic-theme-fixes.config

```diff
@@ -14405,6 +14405,15 @@ body {
 
 ================================
 
+retracmirrors.com
+
+CSS
+div.children {
+    background-color: #101010;
```

Comment:
```suggestion
#app > div {
    background-image: none !important;
```

Reply:
@Gusted thanks for the suggestion! I totally missed the image in the background. I added a new commit, with an additional image I found while editing.